vignettes/User-input-example.Rmd

```diff
@@ -132,21 +136,17 @@ signatures greatly overlap with others.
 `plot_signature_overlap` creates a heatmap of odds ratios. The odds ratio
 represents the odds that two signatures share a specific number of genes.
 
-```{r signature overlap}
-plot_signature_overlap(selected_signatures = active_sigs)
+```{r}
+signature_similarity <- plot_signature_overlap(selected_signatures = active_sigs)
```

Comment:
because you're returning the plotting object [here](https://github.com/greenelab/ADAGEpath/pull/17/files#diff-407f74e9a44e919c1bbcd007dbba7522R229) will this still display the plot?